testapi.pm

```diff
@@ -798,8 +775,7 @@ Returns true if test variable C<$variable> is equal to C<$value> or returns C<un
 
 =cut
 
-sub check_var {
-    my ($var, $val) = @_;
+sub check_var ($var, $val) {
```

Comment:
There are places where only one argument is used (and the uninitialized warning in the log probably ignored ;-)
```
lib/grub_utils.pm
62:    if (check_var("BOOTFROM", "d") && check_var("AUTOUPGRADE") && check_var("PATCH")) {

lib/kdump_utils.pm
216:    assert_script_run('yast2 kdump fadump enable', 180) if check_var('FADUMP');
```

Reply:
I guess I can live with such invalid code causing failures, especially if people ignore warnings.

Reply:
Well, these are the only two places and we can make a PR...

Reply:
Adressed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14997

Reply:
merged, deployed, we can go on.